docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -219,3 +219,14 @@ Note that a standard UDF (non-Pandas) will load timestamp data as Python datetim
 different than a Pandas timestamp. It is recommended to use Pandas time series functionality when
 working with timestamps in `pandas_udf`s to get the best performance, see
 [here](https://pandas.pydata.org/pandas-docs/stable/timeseries.html) for details.
+
+### Compatibiliy Setting for PyArrow >= 0.15.0 and Spark 2.3.x, 2.4.x
+
+Since Arrow 0.15.0, a change in the binary IPC format requires an environment variable to be set in
+Spark so that PySpark maintain compatibility with versions on PyArrow 0.15.0 and above. The following can be added to `conf/spark-env.sh` to use the legacy IPC format:
+
+```
+ARROW_PRE_0_15_IPC_FORMAT=1
```

Comment:
Can we just set it by default?

Reply:
This is for already released Spark versions, where the user has upgraded pyarrow to 0.15.0 in their cluster

Reply:
I think I'd just clarify this in the new docs, that it's only needed if you manually update pyarrow this way.

Reply:
Yes I agree, I'll clarify this